README.md

```diff
@@ -40,6 +40,9 @@ Non-numeric argument to binary operator
 - 🤷‍♀️💰 👍/👎 🍞  
 Missing value where TRUE/FALSE needed
 
+- 💲👩‍♿️⚛️↗️
```

Comment:
How about a different emoji for "invalid", maybe 🤕 or 🙅‍♂️ ?

Reply:
Agree. Much better!